Makefile

```diff
@@ -120,7 +120,7 @@ undeploy: ## Undeploy the PostgreSQL Operator
 
 .PHONY: deploy-dev
 deploy-dev: ## Deploy the PostgreSQL Operator locally
-deploy-dev: PGO_FEATURE_GATES ?= "TablespaceVolumes=true"
+deploy-dev: PGO_FEATURE_GATES ?= "TablespaceVolumes=true,StandalonePGAdmin=false"
```

Comment:
We can leave it unset, but I was thinking it would be nice to have in place at very least while the major dev work is being done. I figured `deploy-dev` was the right place; I didn't do anything for `deploy` itself.

Reply:
why would we need it in place?

Reply:
We don't strictly need it, it's there for convenience. If you don't think it should be there, I can remove it.

Reply:
That makes sense, I think I slightly prefer a comment than being set to off, but I'm good either way

Reply:
After further discussion and the likely name changes, I'll remove this. Thanks for the feedback.